.github/workflows/build-wheel-linux-x86_64.yaml

```diff
@@ -158,7 +158,7 @@ jobs:
 
     - name: Install Dependencies (Python)
       run: |
-        python${{ matrix.python_version }} -m pip install wheel numpy pybind11 PyYAML dataclasses
```

Comment:
Dataclasses became a standard in `Python >=3.7` (ref  https://pypi.org/project/dataclasses/ )  Should we put the standalone version under the `<=3.6` condition somehow?

Reply:
No need, we only support Python 3.8 and above

Reply:
The it should just work without the external dataclasses package.